cal-heatmap/tsconfig.json

```diff
@@ -1,19 +1,20 @@
 {
-    "compilerOptions": {
-        "module": "commonjs",
-        "target": "es6",
-        "noImplicitAny": true,
-        "strictNullChecks": false,
-        "baseUrl": "../",
-        "typeRoots": [
-            "../"
-        ],
-        "types": [],
-        "noEmit": true,
-        "forceConsistentCasingInFileNames": true
-    },
-    "files": [
-        "index.d.ts",
-        "cal-heatmap-tests.ts"
-    ]
-}
\ No newline at end of file
+  "compilerOptions": {
```

Comment:
FYI. The `.editorconfig` I have in my fork specs out indent=2 for json files? So the changes were simply automatically applied.